front/js/components/order.es6

```diff
@@ -12,12 +12,7 @@
     remove: '.js-remove',
     paymentOptions: 'input[name=payment_type]',
     defaultPaymentOptions: 'input[for=id_payment_type_0]',
-    orderForm: {
-      name: '#id_name',
-      phone: '#id_phone',
-      email: '#id_email',
-      city: '#id_city',
-    },
+    orderForm: $('#order-form-full').data('fields'),
```

Comment:
`orderFormData` might be the more good name.
We can't do any actions with this `orderForm`. We can just get it's data